CalligraphySample/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion rootProject.ext.compileSdkVersion
```

Comment:
So you know that as these are in the `allProject` closure, you don't need to define `rootProject.ext.`